app/src/main/java/org/cru/godtools/ui/profile/ProfileActivity.kt

```diff
@@ -0,0 +1,58 @@
+package org.cru.godtools.ui.profile
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.google.android.material.tabs.TabLayoutMediator
+import me.thekey.android.TheKey
+import org.cru.godtools.R
+import org.cru.godtools.activity.BasePlatformActivity
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent.Companion.SCREEN_GLOBAL_DASHBOARD
+import org.cru.godtools.base.ui.activity.BaseActivity
+import org.cru.godtools.databinding.ProfileActivityBinding
+
+fun Activity.startProfileActivity() = startActivity(
+    Intent(this, ProfileActivity::class.java)
+        .putExtras(BaseActivity.buildExtras(this))
+)
+
+class ProfileActivity : BasePlatformActivity() {
+    // region Lifecycle
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ProfileActivityBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        setBindingData()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        mEventBus.post(AnalyticsScreenEvent(SCREEN_GLOBAL_DASHBOARD))
+    }
+    // endregion Lifecycle
+
+    // region Data Binding
+    private lateinit var binding: ProfileActivityBinding
+
+    private fun setBindingData() {
+        val key = TheKey.getInstance(this)
+        binding.accountName = "${key.cachedAttributes.firstName} ${key.cachedAttributes.lastName}"
```

Comment:
you can now get a LiveData handle around the Attributes object using `key.getAttributesLiveData()`. you should utilize this in data binding so that the name is updated when the attributes change.